extensions/ql-vscode/src/model-editor/modeling-store.ts

```diff
@@ -336,6 +342,15 @@ export class ModelingStore extends DisposableObject {
     });
   }
 
+  public addSentToLLMMethods(dbItem: DatabaseItem, sentToLLMMethods: string[]) {
+    this.changeSentToLLMMethods(dbItem, (state) => {
```

Comment:
I decided to follow the existing pattern of having a `changeX` method, even though it's only called from here and we don't have a `removeSentToLLMMethods` method yet. Hopefully that keeps the code more consistent and easier to understand, but let me know if you disagree.